dev/start

```diff
@@ -34,6 +38,30 @@ EOF
   exit
 }
 
+function configure_oidc_providers() {
+  enable_oidc_authenticators
```

Comment:
i would move this call outside so we have
```
configure_oidc_providers
enable_oidc_authenticators
```

Reply:
i would also separate this function into keycloak setups and oidc authenticator setup

Reply:
How? because it is mixed with conjur and keycloak actions?

Reply:
yes